addons/google_calendar/models/google_calendar.py

```diff
@@ -350,118 +348,112 @@ def get_one_event_synchro(self, cr, uid, google_id, context=None):
 
-        if context['curr_attendee']:
-            self.pool['calendar.attendee'].write(cr, uid, [context['curr_attendee']], {'oe_synchro_date': update_date}, context)
+        if self.env.context['curr_attendee']:
+            self.env['calendar.attendee'].browse([self.env.context['curr_attendee']]).write({'oe_synchro_date': update_date})
 
-    def update_an_event(self, cr, uid, event, context=None):
-        data = self.generate_data(cr, uid, event, context=context)
+    def update_an_event(self, event):
+        data = self.generate_data(event)
 
         url = "/calendar/v3/calendars/%s/events/%s" % ('primary', event.google_internal_event_id)
         headers = {}
-        data['access_token'] = self.get_token(cr, uid, context)
+        data['access_token'] = self.get_token()
 
-        status, response, ask_time = self.pool['google.service']._do_request(cr, uid, url, data, headers, type='GET', context=context)
+        status, response, ask_time = self.env['google.service']._do_request(url, data, headers, type='GET')
         #TO_CHECK : , if http fail, no event, do DELETE ?
         return response
 
-    def update_recurrent_event_exclu(self, cr, uid, instance_id, event_ori_google_id, event_new, context=None):
-        gs_pool = self.pool['google.service']
+    def update_recurrent_event_exclu(self, instance_id, event_ori_google_id, event_new):
+        GsPool = self.env['google.service']
 
-        data = self.generate_data(cr, uid, event_new, context=context)
+        data = self.generate_data(event_new)
 
         data['recurringEventId'] = event_ori_google_id
         data['originalStartTime'] = event_new.recurrent_id_date
 
-        url = "/calendar/v3/calendars/%s/events/%s?access_token=%s" % ('primary', instance_id, self.get_token(cr, uid, context))
+        url = "/calendar/v3/calendars/%s/events/%s?access_token=%s" % ('primary', instance_id, self.get_token())
         headers = {'Content-type': 'application/json'}
 
-        data['sequence'] = self.get_sequence(cr, uid, instance_id, context)
+        data['sequence'] = self.get_sequence(instance_id)
 
         data_json = simplejson.dumps(data)
-        return gs_pool._do_request(cr, uid, url, data_json, headers, type='PUT', context=context)
-
-    def update_from_google(self, cr, uid, event, single_event_dict, type, context):
-        if context is None:
-            context = []
-
-        calendar_event = self.pool['calendar.event']
-        res_partner_obj = self.pool['res.partner']
-        calendar_attendee_obj = self.pool['calendar.attendee']
-        calendar_alarm_obj = self.pool['calendar.alarm']
-        user_obj = self.pool['res.users']
-        myPartnerID = user_obj.browse(cr, uid, uid, context).partner_id.id
+        return GsPool._do_request(url, data_json, headers, type='PUT')
+
+    def update_from_google(self, event, single_event_dict, type):
+        CalendarEvent = self.env['calendar.event'].with_context(no_mail_to_attendees=True)
```

Comment:
@WHPODOO with_context({'no_mail_to_attendees': True}) completely replaces the current context. So it is better to use keyword arguments.